packages/shared/src/debug.ts

```diff
@@ -194,3 +194,27 @@ export function useSendPosts(
     };
   }, [channelId, initialDelay, interval, senderRef, session]);
 }
+
+export const useLogChange = (label: string, value: unknown) => {
+  const isFirst = useRef(true);
+  useEffect(() => {
+    if (isFirst.current) {
+      isFirst.current = false;
+      return;
+    }
+    console.log(label, 'changed', value);
```

Comment:
here's a nitpick – neither of my solutions seem worth it to me, but mentioning in case you think of something:

It'd be nice to protect these `console.log`s from adding to noise (e.g. someone commits a `useLogChange`, now other dev needs to either undo that change or comment out this `console.log` to silence).

1. Add third param `logger: ReturnType<typeof createDevLogger>` (although someone can just pass `console` and we'd have same issue of noise)
2. Add lint rule to avoid committing code that calls these functions